src/main/java/edu/cornell/kfs/fp/batch/service/impl/CuDisbursementVoucherDocumentGenerator.java

```diff
@@ -167,6 +178,32 @@ protected void validateAndPopulatePayeeTypeCodeAndPayeeNumber(DisbursementVouche
         payeeDetail.setDisbVchrPayeeEmployeeCode(isEmployee);
         payeeDetail.setDisbursementVoucherPayeeTypeCode(convertedPayeeTypeCode);
         payeeDetail.setDisbVchrPayeeIdNumber(payeeId);
+        payeeDetail.setDisbVchrPayeePersonName(vendorPayeeName);
+    }
+    
+    protected void throwPayeeIdValidationError(String errorMessageKey, String payeeId) { 
+        String employeeErrorMessage = configurationService.getPropertyValueAsString(errorMessageKey);
+        throw new ValidationException(MessageFormat.format(employeeErrorMessage, payeeId));
+    }
+    
+    protected void addWarningMEssageIfPayeeNameNotAccurate(String vendorPayeeName, String dvPayeeName) {
```

Comment:
Minor typo in method name: "addWarningMEssage..." --> "addWarningMessage..."

Reply:
ok